NEWS.md

```diff
@@ -127,6 +127,12 @@
     - In `pl$date_range()`, the argument `end` must be specified and the default
       value of `interval` is changed to `"1d"`. The arguments `start` and `end`
       no longer accept numeric values (#950).
+    - In `pl$scan_parquet()`, the default value of the argument `rechunk` is
+      changed from `TRUE` to `FALSE` (#1033).
+    - In `pl$scan_parquet()` and `pl$read_parquet()`, the argument `parallel`
+      only accepts `"auto"`, `"columns"`, `"row_groups"`, and `"none"` (#1033).
+      Previously, it also accepted upper-case notation of `"auto"`, `"columns"`,
+      `"none"`, and, accepted `"RowGroups"` instead of `"row_groups"`.
```

Comment:
```suggestion
    - In `pl$scan_parquet()` and `pl$read_parquet()`, the argument `parallel`
      only accepts `"auto"`, `"columns"`, `"row_groups"`, and `"none"`.
      Previously, it also accepted upper-case notation of `"auto"`, `"columns"`,
      `"none"`, and `"RowGroups"` instead of `"row_groups"` (#1033).
```